src/libecalc/core/result/results.py

```diff
@@ -4,6 +4,7 @@
 from datetime import datetime
 from typing import Any, Dict, List, Optional, Union
 
+from libecalc.common.stream import Stage
```

Comment:
This should go to `core` (now) or `domain` (later?) project package though, imo. But we will need a representation in the `dto`/application layer as well wrt input/output I guess...

Reply:
Yes, core/domain. Hopefully dto will be allowed to use it from core. It isn't all that clear to me anymore why we need a dto layer..We have a yaml layer, and can convert to something else there, why do we need a layer between yaml and core? I'm sure there is a reason, and I'm thinking it has something to do with evaluated vs not evaluated expressions..

It was introduced to allow changes, but in v2 we can make changes in core. I dunno

Reply:
Let's discuss .)